src/ts/main.ts

```diff
@@ -79,6 +79,7 @@ export { SettingsPanelPageBackButton } from './components/settingspanelpagebackb
 export { SettingsPanelPageOpenButton } from './components/settingspanelpageopenbutton';
 export { SubtitleSettingsPanelPage } from './components/subtitlesettings/subtitlesettingspanelpage';
 export { SettingsPanelItem } from './components/settingspanelitem';
+export { ReplayButton } from './components/replaybutton';
```

Comment:
general question, 

so it would be possible not to `enable` this button, or is it enabled per default? 

Is there some config, or is this done by code?

Reply:
If you mean our default UI: it's not included at all at this point. We merely add and expose the component so that customized UIs can use it easily.